log.py

```diff
@@ -224,6 +219,68 @@ def set_formatter(cls, handler, app_name):
         formatter = JSONFormatter(app_name)
         handler.setFormatter(formatter)
 
+class CloudwatchLogs(Logger):
+
+    NAME = "AWS Cloudwatch Logs"
+
+    GROUP = 'group'
+    STREAM = 'stream'
+    INTERVAL = 'interval'
+
+    DEFAULT_INTERVAL = 60
+
+    SETTINGS = [
+        { "key": GROUP, "label": _("Log Group") },
+        { "key": STREAM, "label": _("Log Stream") },
+        { "key": INTERVAL, "label": _("Update Interval Seconds") },
+    ]
+
+    SITEWIDE = True
+
+    @classmethod
+    def from_configuration(cls, _db, testing=False):
+        settings = None
+        cloudwatch = None
+        from model import (ExternalIntegration, ConfigurationSetting)
```

Comment:
These classes are already imported from model at the top of the file.

Reply:
Good call. Copy pasta on my part. Removing this.